### PR TITLE
Fix #183, Test Cases

### LINKED ISSUE #183: [PE-D][Tester C] bugs: `add` command is case-sensitive
Should the `name` parameter for `add` command NOT be case-sensitive? I believe sometimes the user might forgot whether or not they put the parameters with a capital letter, this could cause some trouble.

### Step to reproduce:
- Make sure you have a clean database (For best results)
- `add urgo 1`: The message will show that you successfully add "urgo" with quantity of 1
- `add Urgo 1`: The message will show that you successfully add "Urgo" with quantity of 1, which should NOT be the case (Notice the capital 'U')

### Expected: 
Item "urgo" will have a quantity of 2

### Actual results:
2 items were created, "urgo" and "Urgo"

### Screenshot:

I believe this could be the case for the other commands as well. If you guys decide to fix this, maybe you should consider making this a program-wide condition.

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: Sukkaito/ped#2

### PR DESCRIPTION
**Pull request by yijiano**
- [x] Closes #183
- [x] Update tP Website
- [x] Update all test cases
- [x] Update PPP

### CODE REVIEW

**Review by cnivedit — APPROVED** [review]
LGTM. Thanks for improving test coverage and for fixing the case sensitivity issues. However, the user guide has a section on case sensitivity which may need to be removed. Additionally, current changes only handle case sensitivity when adding the item to the item map. Consider looking for other instances where case sensitivity might cause issues.

File: src/main/java/seedu/pill/util/ItemMap.java [hunk 1]
```diff
@@ -71,7 +71,7 @@ public boolean equals(Object obj) {
      * @param newItem The item to be added.
      */
     public void addItem(Item newItem) {
-        String name = newItem.getName();
+        String name = newItem.getName().toLowerCase();
```
[the remark this answers is not in the document]
Comment (yijiano) [reply src/main/java/seedu/pill/util/ItemMap.java:74]: Gotcah

File: docs/UserGuide.md [hunk 2]
```diff
@@ -61,15 +61,6 @@ PILL is designed for users who prefer keyboard-based interactions over GUI appli
...
 
-### Case Sensitivity
-
-Our application is **case-sensitive**. This means that item names must match exactly as they were entered. For example:
-
-- `add Panadol` and `add PANADOL` will be treated as two different items.
-- Similarly, commands such as `edit`, `delete` will only work if the case of the item name matches exactly as stored in the inventory.
-
-Ensure you use the correct capitalization when interacting with items in the inventory.
-
```
Comment (cnivedit) [inline docs/UserGuide.md:72]: Good work removing this section

**Review by cnivedit — APPROVED** [review]
LGTM. Thanks for addressing the previous comments

**Pull request merged**